src/services/browse.js

```diff
@@ -17,10 +17,14 @@
 */
 
 import { browseFiles } from "../api/browse";
+import { getLocalStorage } from "../shared/storageHelper";
 
 export function getBrowseData(browseData) {
   return browseFiles(browseData).then((res) => {
-    console.log(res);
-    return res;
+    const data = {
+      res,
+      pages: getLocalStorage("pages"),
```

Comment:
I see, the code is reading pages from Local storage rather than the `X-Total-Pages` header?

Reply:
My bad, it's stored here https://github.com/fossology/FOSSologyUI/pull/81/files#diff-e114ba983ca0ca43d47d5e491d386ff0e442b4483d8b8339160da54f57c4a852R61